verl/utils/checkpoint/fsdp_checkpoint_manager.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         optimizer: Optional[torch.optim.Optimizer] = None,
         lr_scheduler: Optional[torch.optim.lr_scheduler.LRScheduler] = None,
         processing_class: Union[PreTrainedTokenizer, ProcessorMixin] = None,
-        checkpoint_contents: DictConfig = None,
```

Comment:
is this a breaking change? if renaming is desired, we should record in https://github.com/volcengine/verl/discussions/1902